src/com/esotericsoftware/kryo/serializers/FieldSerializer.java

```diff
@@ -602,6 +604,12 @@ public void removeField (CachedField removeField) {
 		return fields;
 	}
 
+    /** Get all transient fields controlled by this FieldSerializer
+     * @return all transient  fields controlled by this FieldSerializer */
+	public CachedField[] getTransientFields () {

```

Comment:
For this use case you can also use `removeField(fieldName)`, which also handles transient fields. Is this sufficient then?